### DISCUSSION TITLE
Hover tooltips?

### DISCUSSION THREAD

**Discussion by ModischFabrications** [arthelokyo/astrowind#381] [Ideas]
I find myself searching for the meaning of some buttons from time to time, especially the RSS button of the template and a few of the lesser known socials. 

GitHub itself uses simple but effective tooltips on hover, as seen below, but there are surely even better examples of them.  
![image](https://github.com/onwidget/astrowind/assets/25404728/09bbefe3-7f7f-4eac-9787-d1f6d9c2664f)

It would be nice to have some pre-build elements, the socials could even reuse the aria-labels there.

**Comment by prototypa** [comment]
Hi @ModischFabrications, thank you for contacting us.

It would be necessary to analyze how many developers may be interested in this and look for a solution that does not affect the performance of the template. **AstroWind** is designed with the philosophy of Simplicity, Best Practices and High Performance and is the initial objective, then each developer can add everything extra they need. Let's evaluate what you propose, thank you.

I'm going to move this to Discussion since it's not a particular issue with **AstroWind**.
